presto-main/src/main/java/com/facebook/presto/Session.java

```diff
@@ -89,6 +90,11 @@ public String getUser()
         return identity.getUser();
     }
 
+    public Optional<Principal> getPrincipal()
```

Comment:
This method is unused and isn't necessary, since we already have a getter for `Identity`.